src/library/scala/util/control/ControlThrowable.scala

```diff
@@ -33,3 +33,33 @@ package util.control
  *  @author Miles Sabin
  */
 trait ControlThrowable extends Throwable with NoStackTrace
+
+object ControlThrowable {
```

Comment:
Is this wrapper object needed?

Reply:
idk. I wanted it to be clear that they're all `ControlThrowable`s